ctapipe/image/tests/test_hillas.py

```diff
@@ -59,7 +59,7 @@ def create_sample_image_selected_pixel(psi="-30d"):
 def compare_result(x, y):
     ux = u.Quantity(x)
     uy = u.Quantity(y)
-    assert isclose(ux.value, uy.value)
+    assert isclose(ux.value, uy.value, equal_nan=True)
```

Comment:
Was there a change in numpy or why is this needed now?

Reply:
no, this should stay. please revert this change.

Reply:
Why did that change?

Reply:
I honestly don't remember....

I will leave it for last to see if I stumble upon the reason why.

Reply:
No, please don't change unit tests for no reason. Please change back and check what and why it breaks.

Reply:
changed back (still, no idea why I did it though, must have been something related to the old code)